app/static/_scss/styles.scss

```diff
@@ -1,10 +1,3 @@
-@use "uswds-core" with (
-    $theme-show-notifications: false,
-    $theme-color-base-light: "gray-cool-5",
-    $theme-color-primary: "blue-70v",
-    $theme-color-primary-vivid: "blue-60v",
-    $theme-color-primary-dark: "blue-warm-70v",
-);
-
+@forward "uswds-theme";
```

Comment:
was there another `uswds-theme` file you meant to include?

Reply:
These are unrelated changes, let me get them out of this PR...